drivers/sensor/mpr/Kconfig

```diff
@@ -0,0 +1,124 @@
+# MPR pressure sensor configuration options
+
+# Copyright (c) 2020 Sven Herrmann
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MPR
+	bool "MPR pressure sensor"
+	depends on I2C && HAS_DTS_I2C
+	help
+	  Enable driver for MPR pressure sensor.
+
+if MPR
+
+choice
+	prompt "MPR Pressure Port"
+	default MPR_PRESSURE_PORT_L
+config MPR_PRESSURE_PORT_L
+	bool "long"
+config MPR_PRESSURE_PORT_S
+	bool "short"
+endchoice
+
+choice
+	prompt "MPR Gel"
+	default MPR_GEL_S
+config MPR_GEL_S
+	bool "silicone"
+config MPR_GEL_F
+	bool "food grade"
+endchoice
+
+choice
+	prompt "MPR Pressure Range"
+	default MPR_PRESSURE_RANGE_0025
+config MPR_PRESSURE_RANGE_0001
+	bool "0 to 1"
+config MPR_PRESSURE_RANGE_01_6
+	bool "0 to 1.6"
+config MPR_PRESSURE_RANGE_02_5
+	bool "0 to 2.5"
+config MPR_PRESSURE_RANGE_0015
+	bool "0 to 15"
+config MPR_PRESSURE_RANGE_0025
+	bool "0 to 25"
+config MPR_PRESSURE_RANGE_0030
+	bool "0 to 30"
+config MPR_PRESSURE_RANGE_0060
+	bool "0 to 60"
+config MPR_PRESSURE_RANGE_0100
+	bool "0 to 100"
+config MPR_PRESSURE_RANGE_0160
+	bool "0 to 160"
+config MPR_PRESSURE_RANGE_0250
+	bool "0 to 250"
+config MPR_PRESSURE_RANGE_0400
+	bool "0 to 400"
+config MPR_PRESSURE_RANGE_0600
+	bool "0 to 600"
+endchoice
+
+choice
+	prompt "MPR Pressure Unit"
+	default MPR_PRESSURE_UNIT_P
+config MPR_PRESSURE_UNIT_P
+	bool "psi"
+config MPR_PRESSURE_UNIT_K
+	bool "kPa"
+config MPR_PRESSURE_UNIT_B
+	bool "bar"
+config MPR_PRESSURE_UNIT_M
+	bool "mbar"
+endchoice
+
+choice
+	prompt "MPR Pressure Reference"
+	default MPR_PRESSURE_REFERENCE_A
+config MPR_PRESSURE_REFERENCE_A
+	bool "absolute"
+config MPR_PRESSURE_REFERENCE_G
+	bool "gage"
+endchoice
+
+choice
+	prompt "MPR Low Pressure"
+	default MPR_LOW_PRESSURE_0000
+config MPR_LOW_PRESSURE_0000
+	bool "0000"
+endchoice
+
+choice
+	prompt "MPR Output Type"
+	default MPR_OUTPUT_TYPE_0
+config MPR_OUTPUT_TYPE_S
+	bool "SPI, not implemented"
+config MPR_OUTPUT_TYPE_0
+	bool "i2c, address 0x08"
+config MPR_OUTPUT_TYPE_1
+	bool "i2c, address 0x18"
+config MPR_OUTPUT_TYPE_2
+	bool "i2c, address 0x28"
+config MPR_OUTPUT_TYPE_3
+	bool "i2c, address 0x38"
+config MPR_OUTPUT_TYPE_4
+	bool "i2c, address 0x48"
+config MPR_OUTPUT_TYPE_5
+	bool "i2c, address 0x58"
+config MPR_OUTPUT_TYPE_6
+	bool "i2c, address 0x68"
+config MPR_OUTPUT_TYPE_7
+	bool "i2c, address 0x78"
```

Comment:
this should come from device tree, not kconfig

Reply:
Ok, removed.